metadata/metadata_test.go

```diff
@@ -2027,3 +2028,247 @@ func TestSourceShallowMapOK(t *testing.T) {
 	assertEqual(t, slices.Contains(sourceVariantResource.Tags, "test.inactive"), true)
 	assertEqual(t, sv.Properties()["test.map.key"], sourceVariantResource.Properties["test.map.key"])
 }
+
+// TODO split these up into better tests
+func Test_GetEquivalent(t *testing.T) {
+	serv, addr := startServNoPanic(t)
+	client := client(t, addr)
+	context := context.Background()
+
+	redisConfig := pc.RedisConfig{
```

Comment:
this could all go into a setup function that the split tests each call

Reply:
yeahhh, there's 100000x better ways to do this lol